reproject/healpix/high_level.py

```diff
@@ -5,7 +5,9 @@
 __all__ = ['reproject_from_healpix', 'reproject_to_healpix']
 
 
-def reproject_from_healpix(input_data, output_projection, shape_out=None, hdu_in=None, order='bilinear', nested=None, field=0):
+def reproject_from_healpix(input_data, output_projection, shape_out=None,
+                           hdu_in=None, order='bilinear', nested=None,
```

Comment:
Should this be changed to 1?

Reply:
`hdu_in=1`?  Maybe.  I got confused which was old and which was new...

Reply:
Yes, ``hdu_in`` should be 1 I think